package.json

```diff
@@ -21,8 +22,8 @@
     "seed": "cd ../sbl-project/dev_setup/mock_data/ && sh create_institutions.sh && sh insert_filing_period.sh",
     "test": "vitest",
     "test:ci": "vitest run",
-    "test:e2e": "yarn preview:test 'cypress open'",
-    "test:e2e:headless": "yarn preview:test 'cypress run'",
+    "test:e2e": "yarn playwright test --ui --workers 4",
+    "test:e2e:headless": "yarn playwright test --workers 4",
     "test:e2e:ci": "vite build && yarn preview:test 'cypress run --record'",
```

Comment:
Keeping these legacy cypress CI tests until we get our playwright tests fully built out.